1-org/modules/centralized-logging/main.tf

```diff
@@ -62,6 +62,12 @@ locals {
     lbk = try(module.destination_logbucket[0].destination_uri, "")
   }
 
+  destination_resource_uri = {
```

Comment:
I can be wrong, but I believe that `destination_resource_uri` map shouldn't have empty string ("") values for certain keys, otherwise the for_each we are consuming this variable will raise an error.

A possible approach to avoid that could be something like that:
``` terraform
filtered_destination_resource_uri = {
    for key, value in destination_resource_uri :
    key => value
    if value != ""
  }
```

Reply:
yes, the values need to be filtered before usage